src/main/resources/schema/wildfly-elytron_1_0.xsd

```diff
@@ -553,19 +573,29 @@
             	</xs:restriction>
         	</xs:simpleType>
         </xs:attribute>
-        <xs:attribute name="password-index" type="xs:int" use="required">
+        <xs:attribute name="password-index" use="required">
             <xs:annotation>
                 <xs:documentation>
                     The column index from an authentication query that represents the user's password.
                 </xs:documentation>
             </xs:annotation>
+            <xs:simpleType>
+                <xs:restriction base="xs:int">
```

Comment:
I am going to go ahead and merge but as this has been used in so many places maybe we could use a single simpleTypeDefinition in the schema that we then use throughout the schema.